bark_ml/environments/single_agent_runtime.py

```diff
@@ -58,18 +58,22 @@ def step(self, action):
     eval_id = self._scenario._eval_agent_ids[0]
     if eval_id in self._world.agents:
       self._world.agents[eval_id].behavior_model.ActionToBehavior(action)
-
+    
     # step and observe
     self._world.Step(self._step_time)
-    observed_world = self._world.Observe([eval_id])[0]
+    observed_world = self._world.Observe([eval_id])
```

Comment:
We should discuss about this

Reply:
This was a design decision of BARK. Would enable to get a list of observed worlds for mult. agent ids.

Reply:
I see the point about the "would enable". But it still remains unclear what to do from our perspective? obviously it did not work with the old setting.